app/views/includes/head.html

```diff
@@ -4,3 +4,5 @@
 {% for stylesheetUrl in extensionConfig.stylesheets %}
   <link href="{{ stylesheetUrl }}" rel="stylesheet" type="text/css" />
 {% endfor %}
+
+<link rel="stylesheet" href="public/stylesheets/accessible-autocomplete.min.css" />
```

Comment:
I would maybe put these third-party scripts/css in a `vendor` directory so we know they're external dependencies.